File: sample-conference-videochat/sample-conference-videochat/Components/ChatScreen/ActionsMenuViewController/MenuActionCell.m
//
//  MenuActionCell.m
//  sample-conference-videochat
//
//  Created by Injoit on 2/6/20.
//  Copyright © 2020 Quickblox. All rights reserved.
//

#import "MenuActionCell.h"

@implementation MenuActionCell

- (void)awakeFromNib {
    [super awakeFromNib];
    
    self.separatorView.hidden = YES;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/ChatScreen/ActionsMenuViewController/MenuActionCell.swift
//
//  MenuActionCell.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 11/22/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

import UIKit

struct MenuActionCellConstant {
   static let reuseIdentifier =  "MenuActionCell"
}

class MenuActionCell: UITableViewCell {

    @IBOutlet weak var actionLabel: UILabel!
    @IBOutlet weak var separatorView: UIView!
    override func awakeFromNib() {
        super.awakeFromNib()
        
        separatorView.isHidden = true
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
}
